### DISCUSSION TITLE
# Knowledge Check

### DISCUSSION THREAD

**Discussion by EY-Mentor3** [github/india#499] [Students]
If somebody plans to migrate several servers from an on-premises network to Azure. What is an advantage of using a public cloud service for the servers over an on-premises network? Choose the correct answer:

1.	The public cloud is owned by the public, NOT a private corporation
2.	The public cloud is a crowd-sourcing solution that provides corporations with the ability to enhance the cloud
3.	All public cloud resources can be freely accessed by every member of the public
4.	The public cloud is a shared entity whereby multiple corporations each use a portion of the resources in the cloud

**Comment by animishpande** [comment]
> If somebody plans to migrate several servers from an on-premises network to Azure. What is an advantage of using a public cloud service for the servers over an on-premises network? Choose the correct answer:
> 
> 1. The public cloud is owned by the public, NOT a private corporation
> 2. The public cloud is a crowd-sourcing solution that provides corporations with the ability to enhance the cloud
> 3. All public cloud resources can be freely accessed by every member of the public
> 4. The public cloud is a shared entity whereby multiple corporations each use a portion of the resources in the cloud

Answer : Option 4
